### PR TITLE
doc/md: improve url doc by adding examples

### PR DESCRIPTION
**Pull request by a8m**

### CODE REVIEW

File: cmd/action/mux.go [hunk 1]
```diff
@@ -89,7 +89,7 @@ func SchemaNameFromURL(url string) (string, error) {
 		return "", err
 	}
 	switch key {
-	case "mysql", "mariadb":
+	case "mysql", "maria", "mariadb":
```
Comment (a8m) [inline cmd/action/mux.go:92]: Exists in "atlas -w" but not in here, so I added it.

**Review by yonidavidson — APPROVED** [review]

File: doc/md/cli/url.mdx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+## SSL/TLS Mode
+
+The default SSL mode for Postgres is `required`. Please follow the 
+[Postgres documentation](https://www.postgresql.org/docs/current/libpq-ssl.html)
+for configuring your SSL connection for your database, or set SSL mode to `disable`
+with the search parameter `?sslmode=disable`. For local databases,
+disabling SSL is appropriate when inspecting and applying schema changes.
+
+MySQL does not require TSL by default. However, you can require TLS
```
Comment (rotemtam) [inline doc/md/cli/url.mdx:85]: Tls

File: doc/md/cli/url.mdx [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+id: cli-url
+slug: /cli/url
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## What is a URL?
+
+A Uniform Resource Locators, or URLs are a standardized method for specifying a location to a resource.
```
Comment (rotemtam) [inline doc/md/cli/url.mdx:12]: Location of a resource

File: doc/md/cli/url.mdx [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+<TabItem value="mysql">
+
+```
+mysql://localhost
+
+mysql://user:pass@localhost
+
+mysql://user:pass@localhost:3306/database
+
+mysql://user:pass@tcp(localhost:3306)/database
```
Comment (rotemtam) [inline doc/md/cli/url.mdx:38]: Technically not a valid url, maybe not mention it?

File: doc/md/cli/url.mdx (hunk 3 above)
Comment (masseelch) [reply doc/md/cli/url.mdx:12]: Drop the A in A Uniform ...

**Pull request merged**